recipes/tvb-data/meta.yaml

```diff
@@ -0,0 +1,39 @@
+
+package:
+  name: tvb-data
+  version: "1.5.4"
+
+source:
+  git_rev: 805f9af4eb05722a52db789307bb81e261ba529a
+  git_url: https://github.com/the-virtual-brain/tvb-data.git
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
```

Comment:
Since `setuptools` is being used, please add `--single-version-externally-managed --record record.txt`.